exercises/hello-world/test/hello_world_test.dart

```diff
@@ -2,21 +2,7 @@ import "package:test/test.dart";
 import "package:hello_world/hello_world.dart";
 
 void main() {
-  var helloWorld = new HelloWorld();
-
-  test("should work", () {
-    helloWorld.hello();
+  test("returns \"Hello, World!\"", () {
+    expect(new HelloWorld().hello(), equals("Hello, World!"));
```

Comment:
Not within the scope of this work. If this is a bug or oversight from previous work, then I think the right step would be to create an issue or PR specificaly to resolve.